dace/runtime/include/dace/reduction.h

```diff
@@ -14,6 +14,11 @@
     #include "../../../external/cub/cub/iterator/transform_input_iterator.cuh"
 #endif
 
+#ifdef __HIPCC__
+// Workaround to define atomic operations on HIP GPU code
```

Comment:
Please elaborate/link stackoverflow thread

Reply:
This is not a workaround, but just defines a macro so that hipcc behaves like nvcc with a new GPU. Reworded comment accordingly.

Reply:
I also changed the whole thing to be more informative macros